.gitignore

```diff
@@ -1,7 +1,7 @@
 .DS_Store
 .idea
-lerna-debug.log
-npm-debug.log
+*.log
+
```

Comment:
`dist/` folder shouldn't be ignored by default, some projects need to check it in